documentation/asciidoc/computers/linux_kernel/patching.adoc

```diff
@@ -1,48 +1,66 @@
-== Patching the kernel
+== Patch the kernel
 
-When xref:linux_kernel.adoc#building[building] your custom kernel you may wish to apply patches, or collections of patches (patchsets), to the Linux kernel.
+When building your custom kernel, you may wish to apply patches or collections of patches (patchsets) to the Linux kernel.
 
-Patchsets are often provided with newer hardware as a temporary measure, before the patches are applied to the upstream (mainline) Linux kernel and then propagated down to the Raspberry Pi kernel sources. However, patchsets for other purposes exist, for instance to enable a fully preemptible kernel for real-time usage.
+We often provide patchsets as a temporary measure to support new hardware before the patches make it into the Linux kernel and the Raspberry Pi kernel. However, patchsets for other purposes exist, for instance to enable a fully pre-emptible kernel for real-time usage.
 
-=== Version identification
+=== Identify your kernel version
 
-It's important to check what version of the kernel you have when downloading and applying patches. In a kernel source directory, running `head Makefile -n 3` will show you the version the sources relate to:
+Always check your version of the kernel before applying patches. In a kernel source directory, run the following command to see the kernel version:
+
+[source,console]
+----
+$ head Makefile -n 3
+----
+
+You should see output similar to the following:
 
-[source]
 ----
 VERSION = 6
 PATCHLEVEL = 1
 SUBLEVEL = 38
 ----
 
-In this instance, the sources are for a 6.1.38 kernel. You can see what version you're running on your system with the `uname -r` command.
+In this instance, the sources are for a 6.1.38 kernel.
+
+To check the kernel version currently running on your device, run the following command:
+
+[source,console]
+----
+$ uname -r
+----
+
+=== Apply patches
 
-=== Applying patches
+Patch application depends on the format used to distribute the patch.
 
-How you apply patches depends on the format in which the patches are made available. Most patches are a single file, and are applied with the `patch` utility. To download and patch our example kernel version with the real-time kernel patches:
+Kernel developers distribute most patches as a single file. Use the `patch` utility to apply these patches. The following commands download, uncompress, and patch our example kernel version with real-time kernel patches:
```

Comment:
Should probably change "Kernel developers" to just "Developers", as not all patchsets come directly from linux maintainers. :shrug: